client/signup/processing-screen/style.scss

```diff
@@ -322,7 +322,7 @@
 }
 
 .signup-processing-screen__loader,
-.signup-processing-screen__loader:after {
+.signup-processing-screen__loader::after {
```

Comment:
This may change the behaviour as `:after` might not always work. Did you test it?

Reply:
Actually you can ignore this:
> Browsers also accept :after, introduced in CSS2.

Reply:
@scruffian When does `:after` and `::after` behave differently? MDN says that the single colon is a CSS2 thing, two colons got introduced in CSS3. My guess would be that the difference was an issue in 2005.